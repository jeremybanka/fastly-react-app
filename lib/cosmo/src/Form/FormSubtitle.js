/* @flow */

import * as React from "react";
import { Box, Text } from "../";

type Props = { children: React.Node };

const FormSubtitle = (props: Props) => (
  <Box>
    <Text fontSize="sm">{props.children}</Text>
  </Box>
);

export default FormSubtitle;
