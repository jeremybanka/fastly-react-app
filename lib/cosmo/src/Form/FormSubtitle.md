The `<Form.Subtitle />` component should be a direct child of `<Form.Header />`.
