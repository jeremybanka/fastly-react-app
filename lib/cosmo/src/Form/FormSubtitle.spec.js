/* @flow */

import React from "react";
import { render } from "enzyme";
import FormSubtitle from "./FormSubtitle";

describe("FormSubtitle test suite", () => {
  it("renders the stored snapshot", () => {
    const component = <FormSubtitle>My Subtitle</FormSubtitle>;
    expect(render(component)).toMatchSnapshot();
  });
});
