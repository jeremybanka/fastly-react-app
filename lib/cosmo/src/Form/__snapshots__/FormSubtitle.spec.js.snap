// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormSubtitle test suite renders the stored snapshot 1`] = `
<div
  class="Box-sc-1v2etov-0 kAhoak"
>
  <span
    class="Text-sc-6c1xd4-0 QgRYE"
    font-size="sm"
  >
    My Subtitle
  </span>
</div>
`;
